api/storage_service.cc

```diff
@@ -1737,12 +1731,6 @@ rest_bind(FuncType func, BindArgs&... args) {
     return std::bind_front(func, std::ref(args)...);
 }
 
-static
-seastar::httpd::future_json_function
-rest_bind(ks_cf_func func, http_context& ctx) {
-    return wrap_ks_cf(ctx, func);
-}
-
```

Comment:
How does the code with this patch compile? You remove the `rest_bind()` function, but continue to use it a few lines down...

Reply:
There are more than one `rest_bind()` overloads in this function. I  patched users of the removed overload into using other overloads that are still there.

Reply:
Oh :-)
Now I remember why function overloading (several different functions with the same name) is my least favorite C++ feature.